src/crate/theme/rtd/crate/layout.html

```diff
@@ -64,6 +74,16 @@
         </div>
         <div class="col-md-8 col-lg-9">
           <div class="wrapper-content-right">
+            {% if display_github %}
+              <div class="view-on-GitHub">
+              {% if check_meta and 'github_url' in meta %}
```

Comment:
You already have this check on line 54, so:

`check_meta and 'github_url' in meta == display_github`

```suggestion
              {% if display_github %}
```

Reply:
but then line 77 & 79 would be again the same

Reply:
Then your checks are wrong. If `display_github` was true, then also `check_meta and 'github_url' in meta` was true, because they are the same.